### PR TITLE
chore: cleanup debris locales

### PR DESCRIPTION
**Pull request by begonaalvarezd**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

Close https://github.com/iotaledger/firefly/issues/7139

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/locales/en.json [hunk 1]
```diff
@@ -1055,15 +1055,6 @@
                 "assetId": "Asset Id"
             }
         },
-        "importErc20Token": {
```
Comment (cpl121) [inline packages/shared/locales/en.json:1058]: Wouldn't it be necessary to remove it from all `locales`?

File: packages/shared/locales/en.json [hunk 2]
```diff
@@ -1603,11 +1594,7 @@
         "jwt": "JSON web token",
         "internalTransaction": "Internal transaction",
         "coinType":"Coin type",
-        "custom": "Custom",
-        "evmAddress": "EVM address",
```
Comment (cpl121) [inline packages/shared/locales/en.json:1607]: filtering by `evmAddress` in the project, I have also seen an interface with the same name `IEvmAddress`, which is not used anywhere. Maybe we could remove it too

**Review by cpl121 — CHANGES_REQUESTED** [review]

File: packages/shared/locales/en.json (hunk 1 above)
Comment (begonaalvarezd) [reply packages/shared/locales/en.json:1058]: we have automation with crowdin, when you touch a locale file you only need to edit the english file and then crowdin takes care of the rest, if we update all locales, crowdin can get confused

File: packages/shared/locales/en.json (hunk 2 above)
Comment (begonaalvarezd) [reply packages/shared/locales/en.json:1607]: Removed in the father PR

**Pull request merged**